features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -153,9 +153,42 @@ const CollapseMenuItem = styled(MenuItemButton)`
 `;
 
 export function SidebarNavigation() {
+  const bodyEl =
+    typeof window !== "undefined" ? document.querySelector("body") : null;
+  const breakpointDesktop = 1024;
   const router = useRouter();
   const { isSidebarCollapsed, toggleSidebar } = useContext(NavigationContext);
+  const [mounted, setMounted] = useState(false);
   const [isMobileMenuOpen, setMobileMenuOpen] = useState(false);
+  const [isMobileScreen, setMobileScreen] = useState(
+    bodyEl != undefined && bodyEl.clientWidth >= breakpointDesktop
+      ? false
+      : true
+  );
```

Comment:
The [matchMedia](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) function might be a better match for this